### PR TITLE
[NUI] Fix gather assembly for xaml

### PR DESCRIPTION
**Pull request by elishateng**
### Description of Change ###

1. Fix gather assembly for xaml

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by wb0716 — APPROVED** [review]
Good for me

**Review by AchoWang — APPROVED** [review]
Looks good to me.

**Review by huayongxu — APPROVED** [review]
lgtm

**Review by AdunFang — APPROVED** [review]
Please fix the compile warning.

**Review by zhouleonlei — APPROVED** [review]
LGTM

**Review by huiyueun — APPROVED** [review]
LGTM

File: src/Tizen.NUI/src/internal/Xaml/XamlParser.cs [hunk 1]
```diff
@@ -319,15 +319,34 @@ static IValueNode GetValueNode(object value, XmlReader reader)
...
+                s_assemblies.Add(typeof(View).GetTypeInfo().Assembly);
+            }
+            else
+            {
+                foreach (var assembly in assemblies)
+                {
+                    try
+                    {
+                        s_assemblies.Add(Assembly.Load(assembly));
+                    }
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/Xaml/XamlParser.cs:344]: could you check the warning at line 345?
there is no action even though an exception occurs.
it seems not good. 
maybe it needs re-throw exception or give an error message.

**Pull request merged**